ui/src/types.js

```diff
@@ -1,3 +1,13 @@
 //@flow
+import { V1Node } from '@kubernetes/client-node';
+
 export type Result<T> = { error: any } | T;
 export type APIResult<T> = Result<{ body: T }>;
+
+export type NodeAddress = {
+  internalIP: string,
+  name: string,
+};
+export type NodesAddresses = Array<NodeAddress>;
```

Comment:
IMO, all the typing here in this file can be removed and type directly in various places.